### ISSUE TITLE
`Fanboy’s Annoyance` blocks duplicate references in the list

### ISSUE THREAD

**Issue by shenzhiming88** [uBlockOrigin/uBlock-issues#1711]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

`Fanboy's Annoyance` block list, `easylist_cookie_specific_uBO.txt` is double referenced twice, I don't know if this is a problem with uBO
In China, I can't open `old.reddit.com`, so I can only send the issue here, sorry!

### A specific URL where the issue occurs

extension://odfafepnkmbhccpbejgmiehpchacaeak/asset-viewer.html?url=fanboy-annoyance

### Steps to Reproduce

- Subscribe to `Fanboy's Annoyance`
- View content

### Expected behavior

`easylist_cookie_specific_uBO.txt` appears only once

### Actual behavior

`easylist_cookie_specific_uBO.txt` appears twice with the same content

![image](https://user-images.githubusercontent.com/66902050/132712315-3e0b67d8-aad8-4eab-8926-0aa2e23bcdb4.png)
![image](https://user-images.githubusercontent.com/66902050/132712352-1816d1e3-22f3-4179-8ab4-de5c04249aa6.png)

### uBlock Origin version

v1.37.0

### Browser name and version

Microsoft Edge 92.0.902.62

### Operating System and version

Windows 10 21H1

**Comment by gorhill** [comment]
It's a filter issue, Fanboy is not maintained by us, please report to <https://github.com/easylist/easylist/issues>.

The uBO-specific sublist is included in both:

- https://github.com/easylist/easylist/blob/master/fanboy-addon/fanboy_annoyance_specific_hide.txt
- https://github.com/easylist/easylist/blob/master/easylist_cookie/easylist_cookie_specific_hide.txt

I gather it should be included only in the second list.

**Comment by shenzhiming88** [comment]
@gorhill 
It seems they don't seem to have settled
Also may I ask, will this exact same filter rule be repeatedly loaded by uBO thus affecting performance? Thanks!

**Comment by gorhill** [comment]
Duplicate filters are discarded, they do not affect memory/CPU footprint. Additionally, the most likely way uBO loads in memory is when there is a valid memory snapshot available, in which case all the work to discard duplicate has been already done and so there is no overhead. But ideally, the issue here should be fixed by Easylist because it's just pointless overhead at parse/compile time.

**Comment by gorhill** [comment]
I am going to investigate whether uBO should mitigate such undue multiple inclusions so as to not depend too much on filter lists from correctly including only once.

**Comment by shenzhiming88** [comment]
Ok, thank you very much for your patience!
The EasyList team may be too busy to work on it
